cogs/Staff.py

```diff
@@ -371,19 +368,9 @@ async def addemoji(
         except ValueError:
             pass
 
-    @commands.slash_command(name="faq")
-    @commands.guild_only()
```

Comment:
why delete...? 


also remove staff check but add channel cooldown of 1m and user cooldown of 5m

Reply:
aren't we going global and this seems very guild specific

Reply:
true true. fair point. but you could always just limit it to the server (only register it in lewis's)

Reply:
thats true we gotta implement a system for that since there are many lewis only commands.

Reply:
altough this command was never used outside of me testing stuff